apps/mull-ui/src/app/pages/login/login.scss

```
@@ -0,0 +1,143 @@
+@import 'variables';
+
+.login-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 1.5em;
+  padding-top: 2.5em;
+  padding-left: 2em;
+  padding-right: 2em;
+}
+
+.facebook {
+  margin-bottom: 2em;
+}
+
+.custom-text-input-container,
+.facebook,
+.google,
+.login,
+.separator,
+.twitter {
+  align-self: center;
+  justify-self: center;
+}
+
+.separator {
+  display: flex;
+  align-items: center;
+  color: $primary-color;
+  width: 100%;
+}
+
+.separator::before,
+.separator::after {
+  content: '';
+  flex: 1;
+  border-bottom: 1px solid $primary-color;
+}
+
+.separator::before {
+  margin-right: 0.25em;
+}
+.separator::after {
+  margin-left: 0.25em;
+}
+
+.login,
+.twitter,
+.google,
+.facebook {
+  width: 100%;
+  height: 45px;
+  cursor: pointer;
+  border-radius: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+}
+
+.twitter,
+.google,
+.facebook {
+  color: #ffffff;
+  font-weight: bold;
+}
+
+.login {
+  font-weight: bold;
+  color: #000000;
+  background-color: rgba(39, 176, 154, 0.6);
+  border: none;
+  outline: none;
+}
+
+.twitter {
+  background-color: $twitter;
+}
+
+.google {
+  background-color: $google;
+}
+
+.facebook {
+  background-color: $facebook;
+  padding-left: 1.2em;
+}
+
+.login-mull-logo {
+  padding-top: 2em;
```

This element's font size is the same as the root element. No need to use em
```suggestion
  padding-top: 2rem;
```

I'll stop leaving comments about em. Just make sure to go over the code and remove instances of em if they don't fit.

You can ask me if you don't know when to use em vs. rem. Short story you most likely want to use rem.